### ISSUE TITLE
2021 Session Updates

### ISSUE THREAD

**Issue by jamesturk** [openstates/issues#161]
Planning for this is coming soon, but wanted to make a catch-all for now:

Master Sheet: https://airtable.com/shrtOIWoL0ddYpe1h -- ask here for access if you need to edit

**Comment by showerst** [comment]
Just a few notes --

Working Prefiles:

- AL
- AR (added 11/16)
- KY (with scraper flag)
- FL
- MO (added 12/3)
- TN
- TX (added 11/16)
- GA (added 11/20)
- OK (added 11/20)
- NV (added 11/20) 

2021 Ignored:

- DE
- ID
- MD
- SC
- WA
- VT

Needs new scraper:

- SD



@jessemortenson ping so you're in the loop as well.

**Comment by jessemortenson** [comment]
Thanks! filed a PR re: KY https://github.com/openstates/task-definitions/pull/9

**Comment by jessemortenson** [comment]
Added a PR for ME. Appears to be the first regular session, starting this wed. :partying_face: 

https://github.com/openstates/openstates-scrapers/pull/3467

**Comment by jessemortenson** [comment]
Say @showerst you have "NM (added 11//24)" listed above but I don't see prefiles on the NM page, nor does the scraper code include a 2021 session, is that a typo?

**Comment by showerst** [comment]
> Say @showerst you have "NM (added 11//24)" listed above but I don't see prefiles on the NM page, nor does the scraper code include a 2021 session, is that a typo?

Oops, yeah that was a special they just ran last week, not a prefile. Apologies, got my wires crossed.

**Comment by showerst** [comment]
ID, VT, and WA all started failing. It looks like none are prefiles though, so i'm just going to queue up the ignores.

**Comment by showerst** [comment]
CA started, working on it now.

**Comment by showerst** [comment]
WA started, working on it today.

**Comment by jessemortenson** [comment]
NH scraper has ingested 4 bills, but they look like bad data. Looks like NH 2021 session posted on source website but it is temporary/template data or something: http://www.gencourt.state.nh.us/bill_status/ - might want to clear those bills in the OS DB and pause the scraper, not sure what protocol is here

**Comment by jessemortenson** [comment]
IA prefiles appear to be vague proposed bills - lacking a clear identifier (maybe the string that is in parentheses? or the PDF filename?). OTOH they do have real bill text. https://www.legis.iowa.gov/legislation/billTracking/prefiledBills?r=true&ga=89&session=1

**Comment by jamesturk** [comment]
Updates so far today:

- DC is up and seems to be working (9 bills)
- NY is up, bills don't have much info yet, but what is there seems to work (lots of blank titled bills being skipped for now)
- MD sponsor fix

I think I have CT working too, verifying now.  Just the FTP listing is down so we can work around it.

**Comment by showerst** [comment]
Just commenting here since slack is down, this afternoon i'll go ahead and queue up the PRs for the rest of this week --

MS, RI, MA, NE, VT

**Comment by yi-yang-github** [comment]
NY data scraper seems to be scraping wrong phone / fax numbers, e.g.:

https://nyassembly.gov/mem/Fred-W-Thiele-Jr/contact/ shows
```
518-455-5997
Fax: 518-455-5963
```
which assumes 518-455-5997 is the phone number.

but the OpenStates data shows phone = 518-455-5963

**Comment by jamesturk** [comment]
@yi-yang-github we have a new issue for that #232

Just waiting on a few states now:

WV: nothing yet, https://www.wvlegislature.gov/Bill_Status/Bill_Status.cfm
~HI: nothing yet~
LA: nothing until April? https://legis.la.gov/legis/BillSearch.aspx?sid=LAST

**Comment by jamesturk** [comment]
LA is up now as well, we're just waiting on WV ⏲️

**Comment by jamesturk** [comment]
WV was just merged!
